Sage/carbon
# feat(font): icon font style and size updated

### Proposed behaviour

Icon font updated with new glyphs and updated the base size of an icon to 20px, with adjustments to the sizes thereafter.

![Screenshot 2022-03-16 at 11 06 17](https://user-images.githubusercontent.com/2411024/158576464-85b50f27-9cb8-47f1-90e8-8cbd2bec2b81.png)

### Current behaviour

The old style Icon font is used with 16px base size.

![Screenshot 2022-03-16 at 11 05 47](https://user-images.githubusercontent.com/2411024/158576493-2980aaa6-ab5f-4ed1-bfce-d07eaf98b5b6.png)


### Checklist


#### QA


### Additional context

n/a

### Testing instructions

While checking the Icon component itself is the base for this update, with the global Icon size change, it therefore impacts anything using an icon and should have a general review of all components.

There is a possible update we need to do with Edit Pod container, but I feel it is not warranted for this PR.
